code_of_conduct.html

```diff
@@ -0,0 +1,128 @@
+      <li><a href="http://cfgmgmtcamp.eu/#route">Location</a></li>
+      <li><a href="http://cfgmgmtcamp.eu/#contributions">Contributions</a></li>
+      <li><a href="code_of_conduct.html">Code of Conduct</a></li>
+      <li><a href="http://cfgmgmtcamp.eu/2014/">2014</a></li>
+    </ul>
+  </nav>
+
+  <div id="container">
+    <header>
+      <div class="inner">
+        <h1>#cfgmgmtcamp</h1>
+        <div class="camp-info">
+          <h2><strong>2</strong> and <strong>3</strong> February 2015</h2>
+          <h2>Gent, Belgium </h2>
+        </div>
+      </div>
+    </header>
+
+
+    <div class="inner">
+      <section id="content">
+        <h1>Code of Conduct</h1>
+
+        <p>
+          Configuration Management Camp is a community conference intended for networking and collaboration in the community of people interested in configuration management.
+        </p>
+
+        <p>
+          We value the participation of each member of the community and want all attendees to have an enjoyable and fulfilling experience. Accordingly, all attendees are expected to show respect and courtesy to other attendees throughout the conference and at all conference events, whether officially sponsored or not.
+        </p>
+
+        <p>
+          Exhibitors, speakers, sponsors, staff and all other attendees at Configuration Management Camp are subject to this Code of Conduct. We are dedicated to providing a harassment-free experience for everyone, and we do not tolerate harassment of participants in any form.
+        </p>
+
+        <p>
+          We ask you to be considerate of others and behave professionally and respectfully to all other participants. Remember that sexual language and imagery is not appropriate for any event venue, including talks. Participants violating these rules may be sanctioned or expelled from the event without a refund at the discretion of the conference organizers.
+        </p>
+
+        <p>
+          Harassment includes offensive verbal comments related to gender, gender identity or expression, sexual orientation, disability, physical appearance, body size, race, religion, sexual images in public spaces, real or implied violence, intimidation, oppression, stalking, following, harassing photography or recording, sustained disruption of talks or other events, inappropriate physical contact, and unwelcome sexual attention. Participants asked to stop any harassing behavior are expected to comply immediately.
+        </p>
+
+        <p>
+          Be careful in the words that you choose. Remember that sexist, racist, and other exclusionary jokes can be offensive to those around you. If you think your conversation is making another community member uncomfortable, try to make amends and move forward.
+        </p>
+
+        <p>
+          If a participant engages in harassing behavior, the event organizers may take any action they deem appropriate, including warning the offender or expulsion from the event with no refund.
```

Comment:
I was thinking that sponsors did pay and may be asked to leave.  They would not get a refund.